### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?

A couple of ``iframe`` bugs fixed:
* Maximizing an ``iframe`` now works.
* We can now resize a SidePanel over an ``iframe``

Other bugs:
* File picker bug fixed. #741 

## Test it please
* Try maximizing minimizing all kind of iframes, navigate away from them, check that everything works as it should.
* Try resizing a SidePanel over an ``iframe`` and see that it works. Try with all kind of iframes. Before it was set a z-index of 1000 for every PersistentIframe. Now that setting is removed, do you think that could cause any issue?

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/web/client/source/class/qxapp/component/widget/NodeView.js [hunk 1]
```diff
@@ -273,6 +274,13 @@ qx.Class.define("qxapp.component.widget.NodeView", {
 
     __attachEventHandlers: function() {
       this.__blocker.addListener("tap", this.__inputNodesLayout.toggleCollapsed.bind(this.__inputNodesLayout));
+
+      qx.event.message.Bus.getInstance().subscribe("maximizeIframe", msg => {
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/component/widget/NodeView.js]: what happens if there are multiple nodes with iframes ? they will all trigger when any of the iframes maximizes

File: services/web/client/source/class/qxapp/desktop/SidePanel.js [hunk 2]
```diff
@@ -82,16 +82,17 @@ qx.Class.define("qxapp.desktop.SidePanel", {
...
+      const splitpaneContainer = this.__getSplitpaneContainer();
       if (collapsed) {
         // Save widths
         this.__savedWidth = this.__getCssWidth();
-        this.__savedMinWidth = this.__getSplitpaneContainer().getMinWidth();
-        this.__getSplitpaneContainer().setMinWidth(0);
-        this.__getSplitpaneContainer().setWidth(20);
+        this.__savedMinWidth = splitpaneContainer.getMinWidth();
+        splitpaneContainer.setMinWidth(0);
+        splitpaneContainer.setWidth(20);
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/desktop/SidePanel.js]:
```
splitpaneContainer.set({
   minWidth: 0,
  width: 20
});

File: services/web/client/source/class/qxapp/desktop/StudyEditor.js [hunk 3]
```diff
@@ -614,6 +615,13 @@ qx.Class.define("qxapp.desktop.StudyEditor", {
 
     __attachEventHandlers: function() {
       this.__blocker.addListener("tap", this.__sidePanel.toggleCollapsed.bind(this.__sidePanel));
+
+      qx.event.message.Bus.getInstance().subscribe("maximizeIframe", msg => {
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/desktop/StudyEditor.js]: all instances will react ... is that intended ?

File: services/web/client/source/class/qxapp/component/widget/NodeView.js (hunk 1 above)
Comment (oetiker) [reply services/web/client/source/class/qxapp/component/widget/NodeView.js]: this problem would not have a visible effect ... I was thinking of the node-input panels ... they exist inside each node, right? and if they all listen to this global message they will act even when the node is not in view ... which seems not an ideal design ...

File: services/web/client/source/class/qxapp/desktop/StudyEditor.js (hunk 3 above)
Comment (oetiker) [reply services/web/client/source/class/qxapp/desktop/StudyEditor.js]: space is great, my only concern is that the bus is global, any iframe maximizing will advertise this to the bus ... all elements will listen to it, regardless if they are currently on screen or not ... could will trigger lots of code which would not be necessary.

File: services/web/client/source/class/qxapp/component/widget/NodeView.js (hunk 1 above)
Comment (ignapas) [reply services/web/client/source/class/qxapp/component/widget/NodeView.js]: Added some code to stop listening to "maximizeIframe" message when the widget is not visible.

File: services/web/client/source/class/qxapp/desktop/StudyEditor.js (hunk 3 above)
Comment (ignapas) [reply services/web/client/source/class/qxapp/desktop/StudyEditor.js]: Corrected!

**Pull request merged**